lib/trento_web/controllers/v1/activity_log_controller.ex

```diff
@@ -11,15 +11,110 @@ defmodule TrentoWeb.V1.ActivityLogController do
   operation :get_activity_log,
     summary: "Fetches the Activity Log entries.",
     tags: ["Platform"],
+    parameters: [
+      first: [
+        in: :query,
+        schema: %OpenApiSpex.Schema{type: :integer},
+        required: false
+      ],
+      last: [
+        in: :query,
+        schema: %OpenApiSpex.Schema{type: :integer},
+        required: false
+      ],
+      after: [
+        in: :query,
+        schema: %OpenApiSpex.Schema{type: :string},
+        required: false
+      ],
+      before: [
+        in: :query,
+        schema: %OpenApiSpex.Schema{type: :string},
+        required: false
+      ],
+      from_date: [
+        in: :query,
+        schema: %OpenApiSpex.Schema{type: :string},
+        required: false
+      ],
+      to_date: [
+        in: :query,
+        schema: %OpenApiSpex.Schema{type: :string},
+        required: false
+      ],
+      actor: [
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Maybe `actors`, as it's a list? No strong opinions, though.

Reply:
I am fine with it as it is, since it can be one or more in number.